### ISSUE TITLE
Translation support

### ISSUE THREAD

**Issue by BraINstinct0** [CMDR-Kiel42/EDMC_SpanshRouter#30]
It would be great to support localization, is there any plan?

**Comment by CMDR-Kiel42** [comment]
Agreed, it would be nice. No plans at the moment, but I'll keep this issue open and update it once it's done
